modules/uid20IdSystem.md

```diff
@@ -0,0 +1,32 @@
+## UID 2.0 User ID Submodule
+
+UID 2.0 ID Module.
+
+### Prebid Params
+
+Individual params may be set for the UID 2.0 Submodule. At least one identifier must be set in the params.
+
+```
+pbjs.setConfig({
+    usersync: {
+        userIds: [{
+            name: 'uid20',
+            storage: {
+                name: 'uid20id',
+                type: 'html5'
+            }
+        }]
+    }
+});
+```
+## Parameter Descriptions for the `usersync` Configuration Section
+The below parameters apply only to the UID 2.0 User ID Module integration.
+
+| Param under usersync.userIds[] | Scope | Type | Description | Example |
+| --- | --- | --- | --- | --- |
+| name | Required | String | ID value for the UID20 module - `"uid20"` | `"uid20"` |
+| storage | Required | Object | The publisher must specify the local storage in which to store the results of the call to get the user ID. This can be either cookie or HTML5 storage. | |
+| storage.type | Required | String | This is where the results of the user ID will be stored. The recommended method is `localStorage` by specifying `html5`. | `"html5"` |
```

Comment:
this all feels out of sync, is this true given your previous comment about a cookie? This says html5 is recommended yet you are saying the other module is requiring a cookie.